packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -43,66 +29,38 @@ const int minimumAndroidSdkVersion = 25;
 /// This should be used over accessing androidSdk.adbPath directly because it
 /// will work for those users who have Android Platform Tools installed but
 /// not the full SDK.
-String getAdbPath([ AndroidSdk existingSdk ]) {
+String getAdbPath(AndroidSdk existingSdk) {
```

Comment:
No - because if we fail to find the android SDK we inject this interface as null, and the intention of this code is to provide a method of looking up adb without the android SDK.


Now, I think we should eventually remove both of those behaviors, but it was too breaking of a change to do at once.